src/pudl/extract/ferc1.py

```diff
@@ -65,6 +67,54 @@
 logger = logging.getLogger(__name__)
 
 
+class Ferc1Datastore(datastore.Datastore):
+    """Provide a thin interface for pulling files from the Datastore."""
+
+    def get_folder(self, year):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/zaneselvans/pudl/pullRequest?prid=5437498)